Droar/Classes/Droar Core/DroarGestureHandler.swift

```diff
@@ -75,8 +75,10 @@ internal extension Droar {
     //Visibility Updates
     private static func beginDroarVisibilityUpdate() -> Bool {
         //Ensure the window is visible
+        let appWindow = loadKeyWindow()
         window.makeKeyAndVisible()
         window.addSubview(navController.view)
+        appWindow?.makeKeyAndVisible()
```

Comment:
Interesting, so what is this doing?  It looks like you're bringing the app window into the foreground in front of Droar

Reply:
No - both windows are visible, we are just switching which the key window is.
The function name is confusing, IMO. I wish there was just a `makeKey()` function for this very reason.